### PR TITLE
CON-2283 track search

### PR DESCRIPTION
**Pull request by Carsten89**
CON-2283 track search

### CODE REVIEW

File: frontend/Plugin.js [hunk 1]
```diff
@@ -62,23 +65,89 @@ class EcondaHessnatur extends SgTrackingPlugin {
...
+    //
+    //this.register.loginFailed(() => {
+    //
+    //});
+
+    let previousSearch = null;
+
+    this.register.search((data) => {
+      // Track a search result page only once
+      if (previousSearch === data.query) {
```
Comment (reneeichhorn) [inline frontend/Plugin.js]: Is this really the wanted behaviour? I'm not sure why this should happen anyway. The comment says it should track the result only once but when searching:
`foo -> bar -> foo`
`foo` is still tracked twice

File: frontend/Plugin.js [hunk 2]
```diff
@@ -62,23 +65,89 @@ class EcondaHessnatur extends SgTrackingPlugin {
...
+        search: [data.query, `${data.hits}`],
+        searchtype: ['Suche - Intern'],
+      };
+
+      this.send(emospro);
+
+      previousSearch = data.query;
+    });
+
+    //this.register.variantSelected(({variant, baseProduct}, raw) => {
```
Comment (reneeichhorn) [inline frontend/Plugin.js]: I know it's WIP but that does not belong her at all ;)

Comment (Carsten89) [reply frontend/Plugin.js]: i know...maybe we should review it only once at the end if everything is done. I waste so much time to create, checkout, switch, merge branches

File: frontend/Plugin.js (hunk 1 above)
Comment (Carsten89) [reply frontend/Plugin.js]: comment was a bit misleading

**Pull request merged**